code/game/turfs/simulated/walls.dm

```diff
@@ -26,7 +26,7 @@
 
 /turf/closed/wall/New()
 	..()
-	builtin_sheet = new sheet_type
+	//builtin_sheet = new sheet_type
```

Comment:
Is there any reason we need to leave this comment in place? If we need it back we can just look into the git history right?